### PR TITLE
feat(plasma-hope): processing unit tests hope

### PR DESCRIPTION
**Pull request by Yakutoc**
Добавлен **Jest** и все что с ним связанно, для запуска **unit-test**.    

Мотивация: 
 
**unit-test** из **plasma-b2c** переехали в  **plasma-hope**. 

:baby_chick: Download canary assets:

[colors--canary.373.4301295867.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.373.4301295867.xml)  
[color_metro_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.373.4301295867.swift)  
[color_metro_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.373.4301295867.kt)  
[color_metro_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.373.4301295867.ts)  
[color_sberHealth_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.373.4301295867.swift)  
[color_sberHealth_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.373.4301295867.kt)  
[color_sberHealth_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.373.4301295867.ts)  
[color_sbermarket_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.373.4301295867.swift)  
[color_sbermarket_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.373.4301295867.kt)  
[color_sbermarket_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.373.4301295867.ts)  
[color_sberprime_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.373.4301295867.swift)  
[color_sberprime_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.373.4301295867.kt)  
[color_sberprime_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.373.4301295867.ts)  
[color_selgros_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.373.4301295867.swift)  
[color_selgros_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.373.4301295867.kt)  
[color_selgros_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.373.4301295867.ts)  
[color_smbusiness_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.373.4301295867.swift)  
[color_smbusiness_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.373.4301295867.kt)  
[color_smbusiness_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.373.4301295867.ts)  
[PlasmaTokensColor--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.373.4301295867.swift)  
[shadow_sbermarket_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.373.4301295867.ts)  
[typo_mage_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.373.4301295867.swift)  
[typo_mage_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.373.4301295867.kt)  
[typo_mage_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.373.4301295867.ts)  
[typo_plasma_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.373.4301295867.swift)  
[typo_plasma_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.373.4301295867.kt)  
[typo_plasma_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.373.4301295867.ts)  
[typo_ruler_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.373.4301295867.swift)  
[typo_ruler_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.373.4301295867.kt)  
[typo_ruler_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.373.4301295867.ts)  
[typo_sage_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.373.4301295867.swift)  
[typo_sage_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.373.4301295867.kt)  
[typo_sage_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.373.4301295867.ts)  
[typo_sbermarket_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.373.4301295867.swift)  
[typo_sbermarket_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.373.4301295867.kt)  
[typo_sbermarket_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.373.4301295867.ts)  
[typo_soulmate_ios-swift--canary.373.4301295867.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.373.4301295867.swift)  
[typo_soulmate_kotlin--canary.373.4301295867.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.373.4301295867.kt)  
[typo_soulmate_react-native--canary.373.4301295867.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.373.4301295867.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.146.0-canary.373.4301295867.0
  npm install @salutejs/plasma-hope@0.5.0-canary.373.4301295867.0
  npm install @salutejs/plasma-web@1.171.0-canary.373.4301295867.0
  # or 
  yarn add @salutejs/plasma-b2c@1.146.0-canary.373.4301295867.0
  yarn add @salutejs/plasma-hope@0.5.0-canary.373.4301295867.0
  yarn add @salutejs/plasma-web@1.171.0-canary.373.4301295867.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-373/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-373/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-373/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-373/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-373/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-373/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-373/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-373/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-373/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-373/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-373/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-373/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-373/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-373/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-373/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-373/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-373/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-373/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-373/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-373/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-373/temple-storybook/

**Review by neretin-trike — APPROVED** [review]

File: packages/plasma-hope/package.json [hunk 1]
```diff
@@ -98,14 +86,9 @@
         "typescript-coverage": "npx typescript-coverage-report > /dev/null"
```
Comment (Yeti-or) [inline packages/plasma-hope/package.json:86]: `test: jest --passWithNoTests` ??

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-373/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-373/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-373/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-373/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-373/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-373/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-373/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Pull request merged**